camel/societies/workforce/workforce_logger.py

```
@@ -0,0 +1,613 @@
+                num_tasks_handled,
+            ) in tasks_handled_by_worker.items():
+                percentage = (
+                    num_tasks_handled / total_tasks_processed_for_utilization
+                ) * 100
+                kpis['worker_utilization'][worker_id_key] = (
+                    f"{percentage:.2f}%"
+                )
+        else:
+            for worker_id_key in (
+                tasks_handled_by_worker.keys()
+            ):  # Ensure all workers who handled tasks are listed, even if 0%
```

Ruff (0.11.9)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)